src/definitions.ts

```diff
@@ -191,12 +191,13 @@ export interface PurchasesPlugin {
 
   /**
    * Removes a given CustomerInfoUpdateListener
-   * @param {CustomerInfoUpdateListener} listenerToRemove CustomerInfoUpdateListener reference of the listener to remove
+   * @param {CustomerInfoUpdateListener} options Include listenerToRemove, which is a CustomerInfoUpdateListener 
+   * reference of the listener to remove
    * @returns Promise with boolean. True if listener was removed, false otherwise
    */
-  removeCustomerInfoUpdateListener(
```

Comment:
We will also need to update the interface in `web.ts`: https://github.com/RevenueCat/purchases-capacitor/blob/a8bb5ff42ca2fbeb33d439dcca0c379163b4d55f/src/web.ts#L70

Additionally, it would be great if you can update the docs in the [README](https://github.com/RevenueCat/purchases-capacitor?tab=readme-ov-file#removecustomerinfoupdatelistener) by using `npm run docgen` or `yarn docgen`

Reply:
done :)